packages/runtime/src/widgets/Collapsible.tsx

```diff
@@ -72,6 +85,10 @@ export type CollapsibleProps = {
   HasItemTemplate;
 
 export const Collapsible: React.FC<CollapsibleProps> = (props) => {
+  const screenContext = useScreenContext();
```

Comment:
Do not use screen context directly. Does the deprecated warning not show up for you?

Reply:
it did, but without this, not able to evaluate the value. I take reference from the chart widget for this

Reply:
Use the useEvaluate hook for this.